htdocs/accountancy/admin/fiscalyear_card.php

```diff
@@ -169,12 +169,12 @@
 
 	// Date start
 	print '<tr><td class="fieldrequired">' . $langs->trans("DateStart") . '</td><td>';
-	print $form->select_date(($date_start ? $date_start : ''), 'fiscalyear');
+	$form->select_date(($date_start ? $date_start : ''), 'fiscalyear');
```

Comment:
I would prefer to use the
print $form->select_date(..... , 1)
with parameter nooutput to 1

Formatting function that include the print must be deprecated to be replaced with the form or function that return the output string.